bin/client/main.rs

```diff
@@ -433,6 +433,19 @@ async fn run() -> std::result::Result<(), ClientError> {
                 }
             }
         }
+        SubCommand::Clean(_clean_opts) => {
+            // Cleans up the local bindles directory.
+            println!("Cleaning up bindles in {:?}...", bindle_dir);
```

Comment:
Nit: It is generally better to just use `{}` and then use the display implementation of `PathBuf`: `bindle_dir.display()`

Reply:
fixed